routers/web/user/profile.go

```diff
@@ -36,6 +36,12 @@ func Profile(ctx *context.Context) {
 		return
 	}
 
+	if strings.Contains(ctx.ContextUser.Name, "@") {
+		ctx.Resp.Header().Add("Location", ctx.ContextUser.Website)
+		ctx.Resp.WriteHeader(http.StatusTemporaryRedirect)
```

Comment:
Please use `ctx.Redirect`

Reply:
This code will be deleted and rewritten soon to render the remote user instead of simply redirecting to their instance.